cvat-ui/src/components/login-with-token/login-with-token.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import { Redirect, useParams } from 'react-router';
+import { useCookies } from 'react-cookie';
+
+export default function LoginWithTokenComponent(){
+    const { sessionId, token } = useParams()
+    const [cookies, setCookie] = useCookies(['sessionid', 'csrftoken'])
+
+    const expires1y = new Date(new Date().setFullYear(new Date().getFullYear() + 1))
+    const expires2w = new Date(new Date().setDate(new Date().getDate() + 13))
+
+    setCookie('sessionid', sessionId, {path: '/', expires: expires2w })
+    setCookie('csrftoken', token, {path: '/', expires: expires1y})
+
+    if ( cookies['sessionid'] && cookies['csrftoken']) {
+        window.location.reload();
+        <Redirect to="/tasks" />
```

Comment:
I suppose redirect after reload is useless. Moreover it should be wrapped into ``return ();``
Maybe something like ``window.location = window.location.origin;`` would be more suitable

Reply:
For some reason. I need to reload window to apply this changes.  However, i have fixed issue with endless reloading when wrong token, by moving this to useEffect